Automattic/vip-go-mu-plugins
# Updating Elasticpress to 3.5.6

## Description

This PR fetches the latest version of our Elasticpress fork, which includes version 3.5.6.

**Please test carefully in different environments, this PR might break things.**

## Changelog Description

### Plugin Updated: Elasticpress 3.5.6

We upgraded Elasticpress to 3.5.6

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Make sure Search is enabled.
3. Perform searches, reindexes, check the Elasticpress admin...
